### ISSUE TITLE
Beam Profiles reverse at some points when used on a Polygon

### ISSUE THREAD

**Issue by tkahng** [hypar-io/Elements#313]
**Describe the bug**
Beam Profiles reverse at some points when used on a Polygon, usually for corners that bend inside.

**To Reproduce**
```
var beam = new Beam(poly, Polygon.Rectangle(input.BridgeWidth, input.BridgeHeight), brdigeMaterial);
```

**Expected behavior**
expected a sweep with profile orientation maintained

**Screenshots**

![image](https://user-images.githubusercontent.com/28710670/81497251-40819400-92f8-11ea-9f7a-d64db273b9b6.png)

**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**
vscode, hypar cli 0.7.1

**Comment by ikeough** [comment]
I can confirm this is still a problem in 0.8. Possible related to #392.
![image](https://user-images.githubusercontent.com/1139788/97642189-c954cf00-1a01-11eb-84db-7978f73b43d1.png)

**Comment by ikeough** [comment]
This is happening in the `Polygon.Frames` method. The concave angle's frame is flipping.

**Comment by ikeough** [comment]
This is fixed in #424
![image](https://user-images.githubusercontent.com/1139788/97644952-e214b300-1a08-11eb-8d25-5ea72f5c4330.png)
